src/app/shared/not-found-page/not-found-page.component.ts

```diff
@@ -1,10 +1,12 @@
 import { Component } from '@angular/core';
+import Constants from 'src/app/constants';
 
 @Component({
 	selector: 'app-not-found-page',
 	templateUrl: './not-found-page.component.html',
 	styleUrls: ['./not-found-page.component.scss'],
 })
 export class NotFoundPageComponent {
+	public images = Constants.assets.images;
```

Comment:
I think `public nofFoundImage = Constants.assets.images.image404;` would be better, you don't need whole images list in this component